locals.tf

```diff
@@ -40,14 +36,14 @@ nginx:
         key: sandbox.gke.io/runtime
         operator: Equal
         value: gvisor
-
 astronomer:
   houston:
     config:
-      commander:
-        enabled: false
-      deployments:
-        logHelmValues: true
+    %{if var.smtp_uri == "true"}
+        email:
+          enabled: true
+          smtpUrl: ${var.smtp_uri == "" ? "" : var.smtp_uri}
+          %{endif}
```

Comment:
this needs ought to be the same indentation level as the opening "if" line - it looks more indented to me. check for tabs vs spaces, I think we only use spaces.